components/content/content-editor/content-editor.ts

```diff
@@ -80,6 +83,7 @@ export default class AppContentEditor extends Vue implements ContentOwner {
 	emojiPanelVisible = false;
 	controlsCollapsed = true;
 	isEmpty = true; // Gets updated through the update-is-empty-plugin
+	openedStartup = false;
```

Comment:
add comment, maybe called it openedOnStartup?

Reply:
actually, I don't know what to call it, because I really don't know what it does...